csvqb/csvqb/models/cube/csvqb/components/attribute.py

```diff
@@ -50,6 +93,16 @@ def validate_data(self, data: PandasDataTypes) -> List[ValidationError]:
         return []  # TODO: implement this
 
 
+@dataclass
+class ExistingQbAttributeLiteral(ExistingQbAttribute, QbAttributeLiteral):
+    def validate_data(self, data: PandasDataTypes) -> List[ValidationError]:
+        errors = []
+
+        errors.append(*[ExistingQbAttribute.validate_data(self, data)])
```

Comment:
`ExistingQbAttribute.validate_data(...)` already returns a list, do you need to wrap it in another one here?

Reply:
Fixed.